subsys/ipc/ipc_service/backends/Kconfig.icmsg

```diff
@@ -13,4 +13,9 @@ config IPC_SERVICE_BACKEND_ICMSG_CB_BUF_SIZE
 	  data bigger than some size, you can safely change this option to
 	  reduce RAM consumption in your application.
 
+# The Icmsg backend in its simplicity requires the system workqueue to execute
+# at a cooperative priority.
+config SYSTEM_WORKQUEUE_PRIORITY
+	range -256 -1
```

Comment:
the default value for `SYSTEM_WORKQUEUE_PRIORITY` is `-1`, why do you need to specify this range here as well?

Reply:
The point is to enforce System workqueue to be cooperative. By default it could be any priority (https://github.com/zephyrproject-rtos/zephyr/blob/main/kernel/Kconfig#L437). I saw this approach in other subsystems, these 4 lines are taken from https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/Kconfig#L42-L45